app/src/components/main/grid/grid.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { GridHeader } from './gridHeader';
+import { GridBody } from './gridBody';
+import { columnPropTypes } from './propTypes';
+import styles from './grid.scss';
+
+const cx = classNames.bind(styles);
+
+export const Grid = ({ columns, data }) => (
+  <div className={cx('grid')}>
+    <GridHeader columns={columns} />
+    <GridBody columns={columns} data={data} />
+  </div>
+);
+Grid.propTypes = {
```

Comment:
Why are using class registration of propTypes and defaultProps  in AbsRelTime. 
We should use common approach

Reply:
TBD